ui/src/app/subnets/views/SubnetDetails/SubnetSummary/SubnetSummaryForm/SubnetSummaryForm.tsx

```diff
@@ -25,80 +25,76 @@ const subnetSummaryFormSchema = Yup.object().shape({
   active_discovery: Yup.boolean(),
   allow_proxy: Yup.boolean(),
   allow_dns: Yup.boolean(),
-  vlan: Yup.number(),
+  vlan: Yup.number().required("VLAN is required"),
 });
 
+type Props = {
+  handleDismiss: () => void;
+  id: Subnet[SubnetMeta.PK];
+};
```

Comment:
@steverydz Typically we define a `Props` type separate from the component - just think it's a bit neater, plus we can export it if we need to.